Rules/UseConsistentIndentation.cs

```diff
@@ -220,16 +220,22 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                 // Check if the current token matches the end of a PipelineAst
                 var matchingPipeLineAstEnd = pipelineAsts.FirstOrDefault(pipelineAst =>
                         PositionIsEqual(pipelineAst.Extent.EndScriptPosition, token.Extent.EndScriptPosition)) as PipelineAst;
-                if (matchingPipeLineAstEnd != null)
+                if (matchingPipeLineAstEnd == null)
```

Comment:
I think some whitespace to break up the logic cascade here might improve readability.
```suggestion

                if (matchingPipeLineAstEnd == null)
```